docs/reference/accessibility/assistive-solutions.md

```diff
@@ -0,0 +1,38 @@
+# Overview of assitive solutions
+
+## Screen readers 
+
+### Orca
+
+Orca is screen reading software for vision impaired and blind users. 
+
+Orca is preinstalled on Ubuntu Desktop and can be enabled and used without any additional configurations. To learn how to tart using Orca, go to [Read the screen aloud](../../how-to/accessibility/read-screen-aloud.md).
+
+### BRLTTY
+
+BRLTTY is a standard service for printing 
+The Orca screen reader can display the user interface on a refreshable Braille display. It uses the BRLTTY service, which provides access to the Linux console for a blind person using a refreshable Braille display. 
+
+As Orca, BRLTTY is preinstalled and can be enabled and used without any additional configurations, see [Read the screen in Braille](../../how-to/accessibility/read-screen-in-braille.md).
+
+## Speech synthethizers
+
+Speech synthethizers are a type of software that are used to convert text to human speech. Typically, they are offered as a backend for integration with other assistive solutions, but often can be used as standalone screen readers.
+
+All of the speech synthethizers in this category are free open-source projects that must be installed on Dekstop manually. 
+
+### eSpeak
+
+eSpeak is a speech synthesizer that supports English. eSpeak can be used as a standalalone tool or as a backend component for screen readers, see [eSpeak website](https://espeak.sourceforge.net/)
+
+### Speech dispatcher
```

Comment:
Maybe worth noting that, as far as I understand, there's a workflow where Orca identifies the text, passes it to Speech Dispatcher, and Speech Dispatcher gives the text to eSpeak to play it as sound.

Therefore, Speech Dispatcher is the layer that allows you to switch between TTS solutions.

BTW, the Arch wiki recommends something called Piper as an alternative to eSpeak and Festival. Could be good wiki content, too:
https://wiki.archlinux.org/title/Speech_dispatcher